jdaviz/configs/default/plugins/gaussian_smooth/gaussian_smooth.py

```diff
@@ -20,6 +21,21 @@
 
 @tray_registry('g-gaussian-smooth', label="Gaussian Smooth")
 class GaussianSmooth(PluginTemplateMixin, DatasetSelectMixin, AddResultsMixin):
+    """
+    See the :ref:`Gaussian Smooth Plugin Documentation <gaussian-smooth>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``dataset`` (:class:`~jdaviz.core.template_mixin.DatasetSelect`):
```

Comment:
As discussed offline, see if the trailing colon is the culprit for gray background. If it is, remove it and maybe try https://github.com/sphinx-doc/sphinx/issues/3778#issuecomment-302936451 if you need multiple lines?
```suggestion
    * ``dataset`` (:class:`~jdaviz.core.template_mixin.DatasetSelect`)
```

Reply:
aaaah, it was the overindenting on the following lines causing an issue (fixed either by forcing multilines with "|" OR by not overindenting so that they're considered a single line).  Thanks!